Formula/jasper.rb

```diff
@@ -1,10 +1,15 @@
 class Jasper < Formula
   desc "Library for manipulating JPEG-2000 images"
-  homepage "https://jasper-software.github.io/jasper/"
-  url "https://github.com/jasper-software/jasper/archive/version-2.0.32.tar.gz"
-  sha256 "a3583a06698a6d6106f2fc413aa42d65d86bedf9a988d60e5cfa38bf72bc64b9"
+  homepage "https://github.com/jasper-software/jasper"
+  url "https://github.com/jasper-software/jasper/releases/download/version-2.0.33/jasper-2.0.33.tar.gz"
+  sha256 "28d28290cc2eaf70c8756d391ed8bcc8ab809a895b9a67ea6e89da23a611801a"
   license "JasPer-2.0"
 
+  livecheck do
+    url :homepage
+    regex(/(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url :stable
    regex(/^version[._-]v?(\d+(?:\.\d+)+)$/i)
```

We prefer to align a check with the `stable` source whenever possible and only fall back to other sources as needed. Both the `homepage` and `stable` URLs are the GitHub repository, so `url :stable` is preferred.

Past that, we almost always anchor the start and end of regexes. Tags are simply brief strings within the `Git` strategy, so we just use `^` and `$` and ensure the regex matches only what we intend. For `jasper`, stable versions use a format like `version-2.0.33`, so we can easily account for this in the regex. The suggested regex here is more in line with how we typically approach `Git` regexes.

Reply:
We should do this in a follow-up PR to avoid re-running CI on this. It takes a few hours, and the CI queue is already backed up.